SalesforcePluginApp.ts

```diff
@@ -0,0 +1,835 @@
+import {
```

Comment:
This is not a good practice adding all of the code in one single file.
Please, look at the [Dialogflow](https://github.com/RocketChat/Apps.Dialogflow/pull/14) for inspiration.

Reply:
Code Refactored 👍